demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime-http.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: serving.kserve.io/v1alpha1
 kind: ServingRuntime
 metadata:
-  name: caikit-tgis-runtime
+  name: caikit-tgis-runtime-http
```

Comment:
Let's leave this one as caikit-tgis-runtime since this is what dashboard uses as their reference, and we'll be HTTP by default

Reply:
caikit-tgis-runtime-http changed to  caikit-tgis-runtime